richtextfx/src/integrationTest/java/org/fxmisc/richtext/api/FxmlTester.java

```diff
@@ -0,0 +1,26 @@
+package org.fxmisc.richtext.api;
+
+import javafx.stage.Stage;
+import javafx.fxml.FXMLLoader;
+import org.fxmisc.richtext.RichTextFXTestBase;
+import org.junit.Test;
+
+public class FxmlTester extends RichTextFXTestBase {
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        // Nothing needed here
+    }
+
+    @Test
+    public void test_fxml_construction_of_area()
+    {
+        Object obj = null;
+        // FxmlTest.fxml is located in resources folder:
+        // src/integrationTest/resources/org/fxmisc/richtext/api/
+        try	{ obj = FXMLLoader.load( getClass().getResource( "FxmlTest.fxml" ) ); }
+        catch ( Exception EX ) { EX.printStackTrace(); }

```

Comment:
There shouldn't be a catch here. Rather, the test method should propagate the thrown exception:
````java
@Test
public void testSomething() throws IOException {
// no longer need the try or catch statements
// try { 
    FXMLLoader.load();
// } catch (IOException EX) {}
````
Also, it should throw the exception that FXMLLoader.load will throw, not the base Exception class upon which all other exceptions are based.

Reply:
Will do ....